app.rb

```diff
@@ -2,6 +2,8 @@
 require './person_class'
 require './student_class'
 require './teacher_class'
+require './create_person'
+require 'json'
 
 class Main
   def initialize()
```

Comment:
- [ ] I will implore you not to make the initialized array in the initialize method empty because the requirements say it should load the whole data if data is present upon the start of the app as attached as image:
![image](https://user-images.githubusercontent.com/86200356/232895891-d6304cd3-666b-4a99-9176-bee5a337716f.png)

This can be achieved by loading the JSON files. Something like:
```
    @books = load_books()
    @people = load_people()
    @rentals = load_rentals()
```